pkg/doc/did/doc.go

```diff
@@ -112,7 +112,12 @@ type PublicKey struct {
 	ID         string
 	Type       string
 	Controller string
-	Value      []byte
+
+	Value []byte
+
+	algorithm string
```

Comment:
this would seem like it could be a publicKeyJWK sub struct?

Reply:
yes, we can just add `publicKeyJwk` field of `*jose.JWK` type.

Reply:
I will also update `verifier.PublicKey` struct.

Reply:
Updated.